### ISSUE TITLE
Committing as a performance

### ISSUE THREAD

**Issue by digital3mpire** [digital3mpire/SUPER-INFORMATION-HIGH-MARKET#5]
Since we do art things and to bring an art related aspect into this i would like to push in the question:
Can committing be seen as a perfarmative act?
(Discussion could also be used as initial content fir a lecture talk on mediaart festivals etc)

**Comment by iintima** [comment]
That's the #wrong question.

**Comment by digital3mpire** [comment]
i am not sure if this really THE WRONG question. BUT as longer i think about it, the more exciting the topic “commit“ appeals to me.
with each commit - an action done ten thousand times a day by coders around the world - these digital superstructure aka digital3mpire is extended a little bit more.

**Comment by digital3mpire** [comment]
committing is becomeing an essential doing in the process of building these new digital controlled reality.

**Comment by kimasendorf** [comment]
The act of commiting is boring, like printing or burning.

**Comment by iintima** [comment]
Sure, of course, but why everything has to be labeled as 'performative'? Nowadays every single shit is already 'performative'. The word lost its meaning. Fancy artytalk. Therefore: yes, committing can be seen as a performative act. Now what? Let's be radical and let's not use that word, pretty please. Committing is too cool to be reduced to 'performative'.

**Comment by digital3mpire** [comment]
true. :-) just wanted to tease and trigger little bit. 4 sure performative really sucks. :-) glad you mentioned it.

**Comment by vommitron** [comment]
"experience economy" vs. _ the economy of attention, : IT'S ALL SPECTACULARly l o n g - - - whilst (#NON the less) [dennoch Langeweile]  seeks our clicks and bates our gaze   {PREVIEW/WINTER  LOOKS/SPRING INTO_FASHION 2018 // REDUCED UP TO 60%}

**Comment by digital3mpire** [comment]
economy attention, by the way, i can see more and more events where you get paid for your attention. earn stuff by watching advertising is one of the most funny new developments
